source/configure/database-configuration-settings.rst

```diff
@@ -388,27 +388,111 @@ Replica lag settings
-|                                                                                                                                           |
-| And for AWS Aurora instances, ``QueryTimeLag`` can be:                                                                                    |
-|                                                                                                                                           |
-| .. code-block:: sql                                                                                                                       |
-|                                                                                                                                           |
-|   select server_id, visibility_lag_in_msec from aurora_global_db_instance_status() where server_id=<>                                     |
-|                                                                                                                                           |
-| For MySQL Group Replication, the absolute lag can be measured from the number of pending transactions in the applier queue:               |
-|                                                                                                                                           |
-| .. code-block:: sql                                                                                                                       |
-|                                                                                                                                           |
-|   select member_id, count_transactions_remote_in_applier_queue FROM performance_schema.replication_group_member_stats where member_id=<>  |
+| - The ``QueryAbsoluteLag`` and ``QueryTimeLag`` queries must return a single row.                                                         |
+| - To properly monitor this you must setup `performance monitoring </scale/performance-monitoring.html>`__ for Mattermost.    |
 +--------------------------------------------------------+----------------------------------------------------------------------------------+
 
+1. Configure the replica lag metric based on your database type. See the following tabs for details on configuring this for each database type.
+
+  .. tabs::
+
+    .. tab:: AWS Aurora
+
+      Add the configuration highlighted below to your ``SqlSettings.ReplicaLagSettings`` array. You only need to add this once because replication statistics for AWS Aurora nodes are visible across all server instances that are members of the cluster. Be sure to change the ``DataSource`` to point to a single node in the group. 
+
+      For more information on Aurora replication stats, see the `AWS Aurora documentaion <https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora_global_db_instance_status.html>`__.
+
+          **Example:**
+
+          .. code-block:: json
+            :emphasize-lines: 4,5,6,7,8
```

Comment:
TIL! I hadn't seen this option before! Thank you!

Reply:
Right?! seemed pretty cool and easy to add.